lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr_context.c

```diff
@@ -86,20 +86,21 @@ ebi_t esm_ebr_context_create(
   ue_mm_context_t* ue_mm_context =
       PARENT_STRUCT(emm_context, struct ue_mm_context_s, emm_context);
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(LOG_NAS_ESM, "ESM-PROC  - ue_mm_context null\n ");
+    OAILOG_ERROR_UE(
+        LOG_NAS_ESM, emm_context->_imsi64, "ESM-PROC  - ue_mm_context null\n ");
     OAILOG_FUNC_RETURN(LOG_NAS_ESM, RETURNerror);
   }
-  OAILOG_INFO(
-      LOG_NAS_ESM,
+  OAILOG_INFO_UE(
+      LOG_NAS_ESM, emm_context->_imsi64,
       "ESM-PROC  - Create new %s EPS bearer context (ebi=%d) "
       "for PDN connection (pid=%d) for ue_id " MME_UE_S1AP_ID_FMT "\n",
       (is_default) ? "default" : "dedicated", ebi, pid,
       ue_mm_context->mme_ue_s1ap_id);
 
   if (pid < MAX_APN_PER_UE) {
     if (ue_mm_context->pdn_contexts[pid] == NULL) {
-      OAILOG_ERROR(
-          LOG_NAS_ESM,
+      OAILOG_ERROR_UE(
+          LOG_NAS_ESM, ue_mm_context->emm_context._imsi64,
```

Comment:
why not using parameter emm_context directly ?

Reply:
Resolved